doc/contents.md

```diff
@@ -12,20 +12,22 @@ Table of contents
     * What is DLite and what is it used for?
     * [Features](features.md)
 2. Installation instructions
-3. [Build instructions](build_with_vs.md)
+3. [Build instructions](build_instructions.md)
 4. Tutorial
 5. Basic topics
     * [Concepts](concepts.md)
     * Data types
     * [Mappings](mappings.md)
     * [Collections](collections.md)
     * Bindings
-6. [Tips and tricks](tips_and_tricks.md)
-7. [Tools](tools.md)
-8. Advanced topics
+    * [Transactions](transactions.md)
+6. [Tools](tools.md)
+7. Advanced topics
     * [Environment variables](environment_variables.md)
-    * [Storage plugins](storage_plugins.md)
-    * Mapping plugins
+    * [Writing storage plugins](storage_plugins.md)
+    * Writing mapping plugins
     * [Code generation](code_generation.md)
-    * [Transactions](transactions.md)
+8. Developer's guide
+    * [Documentation guideline]
```

Comment:
```suggestion
    * [Documentation guideline]
    * [Documentation testing]
```

Reply:
Added "Testing examples in the documentation".